content/svenska/submit.md

```diff
@@ -10,4 +10,22 @@ menu:
         pre: <i class="fas fa-paper-plane"></i>
 ---
 
-Kommer snart.
+Vetenskapsrådet och SciLifeLab har undertecknat Wellcome Trusts ställningstagande angående [Sharing research data and findings relevant to the novel coronavirus (COVID-19) outbreak](https://wellcome.ac.uk/coronavirus-covid-19/open-data), som bland annat anger att undertecknare säkerställer att forskningsresultat och data som har relevans för hanteringen av Coronaviruset snabbt delas och görs öppet tillgängliga:
+ 
+> _"researchers share interim and final research data relating to the outbreak, together with protocols and standards used to collect the data, as rapidly and widely as possible - including with public health and research communities and the WHO"._
+ 
+För att leva upp till detta och göra dina Covid-19-forskningsdata användbara och tillgängliga för resten av forskarsamhället, bör de publiceras i ett publikt repositorie tillsammans med beskrivande metadata.
+Om ditt datamaterial består av datatyper där ett lämpligt internationellt repositorie finns tillgängligt ska det deponeras där, om inget sådant repositorie finns kan ditt datamaterial deponeras i ett [Allmänt repositorie](/support_services/general_data_repository).
```

Comment:
I think the A in Allmänt should not be a capital letter.

Also: is this supposed to become a new page?